src/FabCatalog.js

```diff
@@ -32,10 +33,32 @@ class Fab extends Component {
 const FabDemos = () => (
   <div>
     <h3 className='mdc-typography--subtitle1'>Standard Floating Action Button</h3>
-    <Fab />
+    <Fab ariaLabel='Favorite'>
+      <span className='mdc-fab__icon material-icons'>favorite_border</span>
+    </Fab>
 
     <h3 className='mdc-typography--subtitle1'>Mini Floating Action Button</h3>
-    <Fab mini />
+    <Fab mini ariaLabel='Favorite'>
+      <span className='mdc-fab__icon material-icons'>favorite_border</span>
+    </Fab>
+
+    <h3 className='mdc-typography--subtitle1'>Extended FAB</h3>
+    <Fab extended textLabel='Create'>
+      <span className='mdc-fab__icon material-icons'>add</span>
```

Comment:
the `mdc-fab__icon` should be using an `<i>`

Reply:
Done.